recipe/meta.yaml

```diff
@@ -27,14 +28,23 @@ requirements:
 test:
   imports:
     - powerlaw
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: "http://www.github.com/jeffalstott/powerlaw"
-  license: "MIT"
-  license_family: "MIT"
-  license_file: "powerlaw.py" 
-  summary: "Toolbox for testing if a probability distribution fits a power law"
-  
+  home: https://www.github.com/jeffalstott/powerlaw
+  license: MIT
+  license_family: MIT
+  license_file: powerlaw.py
+  description: |
+    powerlaw is a toolbox using the statistical methods developed in
+    Clauset et al. 2007 and Klaus et al. 2011 to determine if a probability
+    distribution fits a power law.
+  summary: Toolbox for testing if a probability distribution fits a power law
+  doc_url: https://github.com/jeffalstott/powerlaw/blob/master/README.rst
+  dev_url: https://www.github.com/jeffalstott/powerlaw
```

Comment:
can you fix the URL? You should change it to `https://github.com/jeffalstott/powerlaw`so the linter doesn't complain.